lib/blockscore/util.rb

```diff
@@ -65,16 +65,16 @@ def to_constant(camel_cased_word)
     end
 
     def to_camelcase(str)
-      str.split('_').map(&:capitalize).join('')
+      str.split('_').map { |i| i.capitalize }.join('')
     end
 
     # Taken from Rulers: http://git.io/vkWqf
     def to_underscore(str)
-      str.gsub(/::/, '/')
-        .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
-        .gsub(/([a-z\d])([A-Z])/, '\1_\2')
-        .tr('-', '_')
-        .downcase
+      str.gsub(/::/, '/').
+      gsub(/([A-Z]+)([A-Z][a-z])/,'\1_\2').
+      gsub(/([a-z\d])([A-Z])/,'\1_\2').
+      tr("-", "_").
+      downcase
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.